source/_docs/blueprint/selectors.markdown

```diff
@@ -18,13 +18,19 @@ The following selectors are currently available:
 - [Action selector](#action-selector)
 - [Add-on selector](#add-on-selector)
 - [Area selector](#area-selector)
+  - [Example area selectors](#example-area-selectors)
```

Comment:
This is not a TOC, please revert this change

Reply:
Accidental changes restored.